airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/streams/base_insight_streams.py

```diff
@@ -60,7 +60,8 @@ def __init__(
         breakdowns: List[str] = None,
         action_breakdowns: List[str] = None,
         action_breakdowns_allow_empty: bool = False,
-        time_increment: Optional[int] = None,
+        time_increment: Optional[Union[int,str]] = None,
+        date_preset: str = "maximum",
```

Comment:
According to the [FB API doc](https://developers.facebook.com/docs/marketing-api/reference/adgroup/insights/) - `This field is ignored if time_range or time_ranges is specified.`, so I guess this wouldn't work as expected